kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -84,27 +84,22 @@ var ContentAreaView = BaseView.extend({
 
         // Secondly, if the rating_view is previously deleted or never shown before at all, then define it.
         if( typeof this.rating_view === "undefined" ) {
-            this.rating_view = this.add_subview(RatingView, {
-                model: new RatingModel({
-                    "user": window.statusModel.get("user_uri"),
-                    "content_kind": "",
-                    "content_id": ""
-                })
-            });
+            this.rating_view = this.add_subview(RatingView, {});
             this.$("#rating-container-wrapper").append(this.rating_view.el);
         }
 
         // Finally, handle the actual display logic
-        if( this.rating_view.model.get("content_id") !== this.model.get("id") ) {
+        if( this.rating_view.model === null || this.rating_view.model.get("content_id") !== this.model.get("id") ) {
```

Comment:
The idea is that it's always either defined or set to null. From `rating\view.js`:
```js
module.exports = BaseView.extend({

    ...

    initialize: function(options) {
        ...
        this.model = options.model || null;
        ...
    },
```

I got tired of writing `if (typeof foo === "undefined")`, decided that `if( foo === null )` was cleaner. Do you have a preference for this project?